controllers/rhmi/types.go

```diff
@@ -25,13 +25,6 @@ var (
 					},
 				},
 			},
-			{
```

Comment:
@laurafitzgerald just so we won't forget about it - this needs to go back in and be removed in the next release.